app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Post;
+use Illuminate\Http\Request;
+
+class PostController extends Controller
+{
+    public function createpost(Request $request){
+        $incomingFields = $request->validate([
+            'body' => 'required',
+            'title' => 'required'
+        ]);
```

Comment:
Please use FormRequest classes to handle your validations.
https://laravel.com/docs/10.x/validation#form-request-validation